scripts/retdec.py

```diff
@@ -94,9 +95,9 @@ def do_invoke(self, argv):
         fname = "{}/{}.{}".format(path, os.path.basename(params["input_file"]), params["target_language"])
         logfile = "{}/{}.log".format(path, os.path.basename(params["input_file"]))
         if params["mode"] == "bin":
-            cmd = bin_cmd.format(retdec_decompiler, params["mode"], params["raw_endian"], params["architecture"], fname, params["target_language"], params["input_file"])
+            cmd = bin_cmd.format(quote(retdec_decompiler), params["mode"], params["raw_endian"], params["architecture"], fname, params["target_language"], quote(params["input_file"]))
         else:
-            cmd = raw_cmd.format(retdec_decompiler, params["mode"], params["raw_section_vma"], params["raw_entry_point"], params["raw_endian"], params["architecture"], fname, params["target_language"], params["input_file"])
+            cmd = raw_cmd.format(quote(retdec_decompiler), params["mode"], params["raw_section_vma"], params["raw_entry_point"], params["raw_endian"], params["architecture"], fname, params["target_language"], quote(params["input_file"]))
```

Comment:
Just realized that when `decompile -a` is called, the output will be saved in `/tmp/<program_name>` (e.g. `/tmp/hello world`). The file path for this is stored in `fname` so we'll need to wrap that in quotes too.

Reply:
This is why it's better to not use `shell=True`

Reply:
Just made the fix and pushed